openfisca_uk/variables/person/disability.py

```diff
@@ -60,3 +60,38 @@ def formula(person, period, parameters):
             >= parameters(period).benefits.carers_allowance.min_hours
         )
         return already_claiming + meets_requirements
+
+class registered_disabled(Variable):
+    value_type = bool
+    entity = Person
+    label = u'Whether registered disabled'
+    definition_period = YEAR
+
+class dis_equality_act_core(Variable):
+    value_type = bool
+    entity = Person
+    label = u'Whether disabled under the Equality Act (core definition)'
+    definition_period = YEAR
+
+class dis_equality_act_wider(Variable):
+    value_type = bool
+    entity = Person
+    label = u'Whether disabled under the Equality Act (wider definition)'
+    definition_period = YEAR
+
+class is_standard_disabled(Variable):
+    value_type = bool
+    entity = Person
+    label = u'Whether meets a basic level of disability'
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        QUALIFYING = [
+            "is_disabled",
+            "is_severely_disabled",
+            "is_enhanced_disabled",
+            "IIDB",
+            "registered_disabled",
+            "dis_equality_act_core"
+        ]
+        return add(person, period, QUALIFYING, options=[MATCH]) + person.benunit("ESA_income", period, options=[MATCH]) > 0
```

Comment:
Does this mean that all persons in a benefit unit with `ESA_income>0` is classified as disabled?

Also seems like this line might be too long, and lacks a newline.

Reply:
Good catch, I'll take it out

Reply:
Good catch, I'll take it out